proposals/2024-06-14_mixxx_e_v.md

```diff
@@ -0,0 +1,121 @@
+
+## Clarification of Terminology
+
+1. Computer programs considered as free software or "open source software" are put into the public domain free of charge by the originator, allowing others the freedom to use, study, adapt, copy, and improve the program for the common good.
+2. Developing free software involves research, compilation of principles and concepts, and their testing through programming and testing of free software.
+3. "Mixxx" is a DJ software developed as free software and available to the public free of charge.
+
+## 1. Company Name, Registered Office, Financial Year
+
+1. The Association is registered as Mixxx e.V.
+2. The registered offices are located in Berlin. If no fixed office is established, it will be managed from the residence of the Board member acting as general manager.
+3. It is listed under the appropriate register in the District Court’s Register of Associations.
+4. The financial year is the calendar year.
+
+## 2. Purpose of the Association
+
+1. The Association’s purpose is explain the basics in digital DJ-ing to beginners and educate software developemen students and other interested in digital sound processing and online colaboration. For this pupose we promote and distribute free and open source DJ software, particularly "Mixxx". This gives the equality of opportunity in starting with dgital DJ-ing in bedrooms, school parties and up to professional applications. In addition it allows free exchange of knowledge, for education, science and research.
+2. The articles aim to:
+   1. Distribute Mixxx as a free and open source software
+   2. Distribute Manuals and Guides
+   3. Organize user support and education 
+   4. Mentor students during interneships for learning software developmen.
+   5. Maintaining the development infrstructur for easy code access
+3. Amendments to the association’s purpose may only be made in accordance with § 3 (1).
+
+## 3. Non-profit Association
+
+1. The Association’s activities are exclusively non-profit and tax-privileged in accordance with German Tax Regulations. Its activities are for the common good and not for profit-making.
+2. Financial means may only be used for purposes stipulated in the Articles of Association. Members are reimbursed for expenses but do not receive direct benefits from the Association’s funds.
+3. It is not permitted to favor someone through association expenditures that do not serve the association's purpose or are disproportionately high.
+
+## 4. Types of Membership and Members of the Association
+
+1. Membership is open to natural persons and legal bodies supporting the association's aims. Membership types include:
+   1. Active members: Natural persons actively supporting the Association’s purpose and aims.
+   2. Extraordinary members: Natural persons and legal bodies supporting the Association’s aims but without voting rights.
+   3. Supporting Members: Extraordinary members primarily supporting through financial or material contributions.
```

Comment:
Let's use "Ordinary Member" (de_DE: "ordentliches Mitglied"), it's less ambiguous. Also, 'ets clarify the difference between 2 and 3:

```suggestion
   1. Ordinary members: Natural persons supporting the Association’s purpose and aims with full voting rights.
   2. Extraordinary members: Natural persons and legal bodies supporting the Association’s aims. These members have the right to participate in the General Assembly, but do not have voting rights.
   3. Supporting Members: Extraordinary members primarily supporting through financial or material contributions, without the right to participate in the General Assembly.
```